### PR TITLE
Add sketchup.com

### PR DESCRIPTION
**Pull request by Yang-Nankai**
SketchUp is a 3D modeling software used for creating architectural, interior design, and engineering models. It is popular among architects, designers, and engineers for its ease of use and powerful design tools. The platform provides 2FA for the account login verification process. The website is ranked 2,724 on [Similarweb](https://pro.similarweb.com/#/digitalsuite/websiteanalysis/overview/website-performance/*/999/1m?webSource=Total&key=sketchup.com). The website provides documentation on how to turn on 2FA.

### CODE REVIEW

**Comment by Yang-Nankai** [comment]
After my test, I found that except for **passkey**, which can be logged in without a password, **totp** and **secondary email** both require a password and a one-time code to log in. Here are some screenshots:

70ab324a1d30a965d59f67d1724f4f9

**Comment by Yang-Nankai** [comment]
image

**Comment by hkamran80** [comment]
> except for **passkey**, which can be logged in without a password

You can make a PR to the [Passkeys Directory](https://github.com/2factorauth/passkeys) for it (if you want to)!

File: entries/s/sketchup.com.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "SketchUp": {
+    "domain": "sketchup.com",
+    "tfa": [
+      "email",
+      "totp"
+    ],
+    "documentation": "http://help.sketchup.com/en/admin/multi-factor-authentication",
```
Comment (hkamran80) [inline entries/s/sketchup.com.json]:
```suggestion
    "documentation": "https://help.sketchup.com/admin/multi-factor-authentication",
```

File: img/s/sketchup.com.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg version="1.1" id="prefix__prefix__Layer_1" xmlns="http://www.w3.org/2000/svg" x="0" y="0" viewBox="0 0 100 100" width="200" height="200" xml:space="preserve"><style>.prefix__prefix__st1{fill:#005f9e}</style><path class="prefix__prefix__st1" d="M100.1 24.02L64.69 3.61S59.08.33 50.13.33c-9.32 0-14.72 3.38-14.72 3.38L2.61 22.48l49.9 28.68v57.96l32.84-18.75s5.87-3.24 10.33-10.94c3.93-6.78 4.42-13.34 4.42-13.34V24.02zm-53.4 9.44s7.32-1.35 13.51 2.22c6.19 3.57 9.84 5.67 9.84 5.67s.01 4.97.01 12.4c0 5.95-5.04 12.12-5.04 12.12l.01-21.83L46.7 33.46zm29.02 47.88l-10.69 6.13v-5.75l8.32-4.76c7.92-5.71 9.23-14.13 9.23-17.05l-.03-25.86-22.69-13.06s-3.45-2.26-9.65-2.26-9.58 2.09-9.58 2.09l-7.97 4.58-5.03-2.91 11.01-6.29s4.31-2.39 11.53-2.39c6.63 0 11.79 2.59 11.79 2.59l25.62 14.8-.03 28.22c.18 4.34-1.84 14.94-11.83 21.92z"/><path class="prefix__prefix__st1" d="M.09 41.25l34.98 20.07-.03 5.98-19.9-11.41-12.52 7.12 32.4 18.67.02 5.76L.33 67.47c.09 2.34 2.21 9.55 6.27 15.13 4.12 5.65 8.82 8.05 8.82 8.05l32.08 18.47V54.04L.12 26.83.09 41.25z"/></svg>
```
Comment (hkamran80) [inline img/s/sketchup.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100"><path d="M100.1 24 64.7 3.6S59 .3 50 .3c-9.3 0-14.7 3.4-14.7 3.4L2.6 22.5l50 28.7v58l32.8-18.8s5.8-3.3 10.3-11c4-6.7 4.4-13.3 4.4-13.3V24zm-53.4 9.5S54 32 60.2 35.7l9.8 5.6v12.5c0 5.9-5 12-5 12V44L46.7 33.5zm29 47.8L65 87.5v-5.8l8.4-4.7c7.9-5.8 9.2-14.2 9.2-17V34L59.8 21s-3.5-2.3-9.7-2.3-9.6 2.1-9.6 2.1l-8 4.6-5-3 11-6.2s4.3-2.4 11.6-2.4A29 29 0 0 1 62 16.4l25.6 14.8v28.2c.1 4.4-1.9 15-11.9 22z" fill="#005f9e"/><path d="m0 41.3 35 20v6L15.1 55.9 2.6 63 35 81.7v5.7L.3 67.4c.1 2.4 2.2 9.6 6.3 15.2 4.1 5.7 8.8 8 8.8 8l32.1 18.5v-55L.1 26.7v14.4z" fill="#005f9e"/></svg>
```

**Review by hkamran80 — CHANGES_REQUESTED** [review]

File: entries/s/sketchup.com.json (hunk 1 above)
Comment (Yang-Nankai) [reply entries/s/sketchup.com.json]: Thank you for your correction!

**Review by hkamran80 — APPROVED** [review]

**Comment by hkamran80** [comment]
Thank you! Your contribution will be visible in around 10 minutes, although it may take longer due to [server and browser caching](https://github.com/2factorauth/twofactorauth/wiki/FAQ-Caching).

**Pull request merged**